src/Symfony/Component/Mailer/Bridge/Mailgun/Mailgun.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Bridge\Mailgun;
+
+/**
+ * @author Michael Garifullin <garifullin@gmail.com>
+ *
+ * @experimental in 4.3
+ */
+class Mailgun
```

Comment:
Can we find a more expressive name? `MailgunRegions` maybe?

Reply:
me too confused with that naming